aws/resource_aws_cognito_user_pool_test.go

```diff
@@ -108,9 +108,9 @@ func TestAccAWSCognitoUserPool_basic(t *testing.T) {
 				Check: resource.ComposeAggregateTestCheckFunc(
 					testAccCheckAWSCognitoUserPoolExists("aws_cognito_user_pool.pool"),
 					resource.TestMatchResourceAttr("aws_cognito_user_pool.pool", "arn",
-						regexp.MustCompile("^arn:aws:cognito-idp:[^:]+:[0-9]{12}:userpool/[\\w-]+_[0-9a-zA-Z]+$")),
+						regexp.MustCompile(`^arn:aws:cognito-idp:[^:]+:[0-9]{12}:userpool/[\\w-]+_[0-9a-zA-Z]+$`)),
```

Comment:
When switching from interpreted string literals to raw string literals, you need to remove the escaping for `\`. This currently fails acceptance testing:

```
--- FAIL: TestAccAWSCognitoUserPool_basic (4.72s)
    testing.go:538: Step 0 error: Check failed: 2 errors occurred:
        	* Check 2/6 error: aws_cognito_user_pool.pool: Attribute 'arn' didn't match "^arn:aws:cognito-idp:[^:]+:[0-9]{12}:userpool/[\\\\w-]+_[0-9a-zA-Z]+$", got "arn:aws:cognito-idp:us-west-2:*******:userpool/us-west-2_YYLNjiL0j"
        	* Check 3/6 error: aws_cognito_user_pool.pool: Attribute 'endpoint' didn't match "^cognito-idp\\\\.[^.]+\\\\.amazonaws.com/[\\\\w-]+_[0-9a-zA-Z]+$", got "cognito-idp.us-west-2.amazonaws.com/us-west-2_YYLNjiL0j"
```

```suggestion
						regexp.MustCompile(`^arn:aws:cognito-idp:[^:]+:[0-9]{12}:userpool/[\w-]+_[0-9a-zA-Z]+$`)),
```